src/fontra/views/editor/editor.js

```diff
@@ -229,7 +229,7 @@ export class EditorController {
       rootSubscriptionPattern[rootKey] = null;
     }
     await this.fontController.subscribeChanges(rootSubscriptionPattern, false);
-    await this.initGlyphsSearch();
+    this.getSidebarPanel("glyph-search").setup();
```

Comment:
Can't this be done as part of the constructor of the panel? Or as part of the attach method?